test/mam_jid_mini_SUITE.erl

```diff
@@ -0,0 +1,32 @@
+-module(mam_jid_mini_SUITE).
+-include_lib("eunit/include/eunit.hrl").
+-compile([export_all]).
+
+-define(ae(Expected, Actual), ?assertEqual(Expected, Actual)).
+
+all() -> [test_encode_decode_functionality].
+
+test_encode_decode_functionality(_Config) ->
```

Comment:
(Valid only if eunit is used) `encode_decode_functionality_test(...` seems to be the naming convention based on the examples from http://erlang.org/doc/apps/eunit/chapter.html#Getting_started